### PR TITLE
Hw 2

### PR DESCRIPTION
**Pull request by lhetstest**

### CODE REVIEW

File: awscli/VPC.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    --key-name web-key \
+    --security-group-ids sg-00072bba1a5bec5cd \
+    --subnet-id subnet-060113e650ffd523a \
+    --tag-specifications 'ResourceType=instance,Tags=[{Key=Name,Value=FRONT},{Key=Environment,Value=DEV}]' 
+
+## task: connect with ssh db-host
+
+# прописати по хостах в конфіг at local .ssh
+
+Host front
```
Comment (saaverdo) [inline awscli/VPC.md:93]: In markdown you have to end each line with double space or they'll be treated as one line, not separate ones.

File: awscli/ip-update.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+echo "Host $DB_HOST_ALIAS" >> $TEMP_CONFIG
+echo "  HostName $PRIVATE_IP" >> $TEMP_CONFIG
+echo "  User ubuntu" >> $TEMP_CONFIG
+echo "  IdentityFile ~/.ssh/back-key.pem" >> $TEMP_CONFIG
+echo "  ProxyJump $JUMP_HOST_ALIAS" >> $TEMP_CONFIG
+
+echo "" >> $TEMP_CONFIG
+
+# Переміщення нового конфігу на місце старого
+mv $TEMP_CONFIG $SSH_CONFIG
```
Comment (saaverdo) [inline awscli/ip-update.sh:71]: In scope of homework it's ok.
But if you already have an ssh config with other records, they'll be lost.

**Review by saaverdo — APPROVED** [review]
Good.
Approved)

File: awscli/ip-update.sh (hunk 2 above)
Comment (lhetstest) [reply awscli/ip-update.sh:71]: Is there any possibility of updating only one row?

**Pull request merged**